cmds/core/sluinit/uinit.go

```diff
@@ -73,13 +75,38 @@ func main() {
 	}
 	slaunch.Debug("Collectors completed")
 
-	slaunch.Debug("********Step 4: Write eventlog to /boot partition*********")
-	if e := p.EventLog.Persist(); e != nil {
-		log.Printf("EventLog.Persist() failed err=%v", e)
-		os.Exit(1)
+	slaunch.Debug("********Step 4: Measuring target kernel, initrd ********")
+	if e := p.Launcher.MeasureKernel(tpmDev); e != nil {
```

Comment:
I know e works but the common idiom is err, if you are ok with that change.

Reply:
@rminnich We were almost ready to merge this.. So any change at this point would mean retesting/rebasing on real hardware..I will wait for you to review the rest of this PR and if there is any other major change, I can club this along with it, Otherwise I will mark it for next PR.

Reply:
Ah never mind, I had to rebase so i made these changes....plz ignore my previous comment..